templates/CMakeLists.txt.template

```diff
@@ -358,6 +362,7 @@
     PRIVATE <%text>${_gRPC_BENCHMARK_INCLUDE_DIR}</%text>
     PRIVATE <%text>${_gRPC_CARES_INCLUDE_DIR}</%text>
     PRIVATE <%text>${_gRPC_GFLAGS_INCLUDE_DIR}</%text>
+    PRIVATE <%text>${CMAKE_CURRENT_BINARY_DIR}</%text>/third_party/address_sorting
```

Comment:
should use _gRPC_ADDRESS_SORTING_INCLUDE_DIR just as the other third_party deps do. Definitions goes into cmake/address_sorting.cmake